### DISCUSSION TITLE
Import Error : NameError: name '_____' is not defined  on Visual Studio Code Capstone Flask Dashboard UI

### DISCUSSION THREAD

**Discussion by andhikahermawan** [teamalgoritma/community#196] [Q&A] [answered]
Tolong petunjuknya saat app.py mau di running tulisannya error name is not defined.
error line 10, error import csv padahal data csv tidak ada yang diubah letak maupun filenya

![tanya github 3](https://user-images.githubusercontent.com/107476066/178157714-788a3934-6913-4afe-8f3e-1632d113f089.png)
.
![tanya github 4](https://user-images.githubusercontent.com/107476066/178157825-0496815d-8c38-49b9-852f-cb4f0a824fb6.png)

**Comment by triarm** [comment, marked as answer]
Hi @andhikahermawan ,
Dilihat dari error tersebut, sepertinya Anda belum menyimpan ulang (save) script app.py setelah melakukan pengeditan. Silahkan untuk lakukan saving kemudian jalankan kembali script app.py tersebut.

Semoga menjawab
